indico/modules/events/editing/client/js/components/Timeline/judgment/AcceptRejectForm.jsx

```diff
@@ -48,7 +51,7 @@ export default function AcceptRejectForm({action, setLoading}) {
               /* otherwise changing required doesn't work properly if the field has been touched */
               key={action}
             />
-            <div>TODO: Tags field</div>
+            <FinalTagInput name="tags" options={_.sortBy(tagOptions, 'id')} />
```

Comment:
I'd do the sorting inside the field

Reply:
Why?

Reply:
we probably always want to show it sorted (by code btw, not by id!), so having to do it whenever you use the field is a bit repetitive

Reply:
another option would be just returning it sorted from the API (in the schema's post_dump function)... natural-order sorting would be easier there as well

Reply:
Yeah, I think I prefer that last option.